assembl/graphql/schema.py

```diff
@@ -1432,8 +1432,15 @@ def mutate(root, args, context, info):
         original_subject_entry = post.subject.first_original()
         # subject is not required, be careful to not remove it if not specified
         if subject and subject != original_subject_entry.value:
-            post.subject.add_value(subject, original_subject_entry.locale_code)
             changed = True
+            post.subject.add_value(subject, original_subject_entry.locale_code)
+            # Edit subject for all descendants
+            children = post.children[:]
+            new_subject = u'Re: ' + restrip_pat.sub('', subject).strip()
+            while children:
+                child = children.pop()
+                children.extend(child.children)
```

Comment:
is it ok to modify `children` while you are looping through it ? I would have done a recursive function for this kind of stuff.

Reply:
I made a copy, see `post.children[:]`